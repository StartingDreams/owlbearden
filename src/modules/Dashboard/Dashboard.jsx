import React from 'react';
import injectTapEventPlugin from 'react-tap-event-plugin';

injectTapEventPlugin();

const Dashboard = () => (
  <div>
    <h1>Dashboard</h1>
  </div>
);

export default Dashboard;
