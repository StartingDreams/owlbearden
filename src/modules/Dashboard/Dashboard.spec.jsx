import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Dashboard } from './Dashboard';

describe('<Init />', () => {
  it('spell should match the snapshot', () => {
    const wrapper = shallow(<Dashboard />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
