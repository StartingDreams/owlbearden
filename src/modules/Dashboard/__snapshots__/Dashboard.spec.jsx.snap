// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Init /> spell should match the snapshot 1`] = `
<Card
  expandable={false}
  expanded={null}
  initiallyExpanded={false}
>
  <CardTitle
    subtitle="Spell helpers"
    title="Casting"
  />
  <CardText>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec mattis pretium massa. Aliquam erat volutpat. Nulla facilisi. Donec vulputate interdum sollicitudin. Nunc lacinia auctor quam sed pellentesque. Aliquam dui mauris, mattis quis lacus id, pellentesque lobortis odio.
  </CardText>
  <CardActions>
    <FlatButton
      containerElement={
        <NavLink
          activeClassName="active"
          to="/casting/spells"
        />
      }
      disabled={false}
      fullWidth={false}
      icon={
        <FontIcon
          className="material-icons"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          whatshot
        </FontIcon>
      }
      label="Spells"
      labelPosition="after"
      labelStyle={Object {}}
      onKeyboardFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
      primary={false}
      secondary={false}
    />
  </CardActions>
</Card>
`;
